src/libs/zmbv/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_library(zmbv STATIC zmbv.cpp)
+
+pkg_check_modules(ZLIB_NG REQUIRED IMPORTED_TARGET zlib-ng)
+
+target_include_directories(zmbv PUBLIC ..)
+# FIXME: "ZLIB_NG_LIBRARIES" points to the wrong library name (z-ng), use this simpler way when the port is fixed
```

Comment:
VCPKG has fixed this issue, although I haven't tried it yet, so I'll update this ASAP. Only "issue" is that I need to bring the vcpkg baseline to a very recent commit, just need "approval" from any of you to do so.

Reply:
Is the 2024.05.24 vcpkg new enough? I've just submitted #3761 which bumps the baseline to that release.

Reply:
Probably not, we need a baseline from yesterday.

Reply:
@Arcnor Are you waiting for us to push the baseline forward to be able to finish this PR?

Reply:
@Arcnor If you would be able to do that, that would be great, thanks.

Reply:
I've updated the baseline to one from yesterday. IT's maybe too recent, you let me know, otherwise I can pick one from 9 days ago.